buildSrc/build.gradle

```diff
@@ -152,6 +156,18 @@ if (project != rootProject) {
   jarHell.enabled = false
   thirdPartyAudit.enabled = false
 
+  // tests can't  be run with randomized test runner
+  // it's fine as we run them as part of :buildSrc
+  test.enabled = false
+  task integTest(type: Test) {
+    exclude "**/*Tests.class"
+    include "**/*IT.class"
+    testClassesDirs = sourceSets.test.output.classesDirs
+    classpath = sourceSets.test.runtimeClasspath
+    inputs.dir(file("src/testKit"))
```

Comment:
why is this necessary now?

Reply:
This doesn't really belong here, it got pulled in as I tried to separate this PR out, and forgot to remove it.  The  integration tests use these sample Gradle projects. What this line does is makes sure tests are not considered up to date if only this directory changes.

Reply:
ok